### PR TITLE
net-nntp/slrn: add missing include

### PR DESCRIPTION
**Pull request by NHOrus**
Closes: https://bugs.gentoo.org/874687

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: net-nntp/slrn/files/slrn-1.0.3-configure.patch [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Subject: [PATCH] autoconf: Fix C99 compatibility issue in the VA_COPY checks
+
+The exit function is called without including <stdlib.h>, resulting
+in an implicit function declarations.  Future compilers will not
+support implicit function declarations by default.  This will lead
+to build failures.
+---
+ autoconf/configure.ac | 3 +++
+ configure             | 3 +++
+ 2 files changed, 6 insertions(+)
```
Comment (juippis) [inline net-nntp/slrn/files/slrn-1.0.3-configure.patch:18]: Hmm, you should pick one to patch. If patching **configure.ac** then remember to inherit *autotools.eclass* and call `eautoreconf` in `src_prepare()`.

**Review by juippis — COMMENTED** [review]
Note to self: this is most likely revbumped due to not breaking stable.

**Comment by NHOrus** [comment]
It's revbumped, maybe I shouldn't revbump, dunno.
To run autotools, this needs to be patched more, because automake.in is full of definitions, and automake.am is just weird and mostly empty, so eautoreconf breaks.

File: net-nntp/slrn/files/slrn-1.0.3-configure.patch (hunk 1 above)
Comment (eli-schwartz) [reply net-nntp/slrn/files/slrn-1.0.3-configure.patch:19]: Upstream is one of those fascinating projects that:
- checks configure scripts into git
- distributes source code that *cannot* be `autoreconf`ed
- feels it is "unclean" to have configure.ac next to configure, so shoves it off into a subdir, which doesn't autoreconf either, but does autoconf.


I don't know that autotools.eclass can handle *any* of this.

**Comment by eli-schwartz** [comment]
```diff
-KEYWORDS="amd64 ppc x86 ~amd64-linux ~x86-linux"
+KEYWORDS="~amd64 ~x86"
```

Is there a specific reason we dropped ppc here?

**Pull request closed without merging**